dwhdelft.nl/components/pages/resources/PurpleFriday.vue

```diff
@@ -0,0 +1,62 @@
+<i18n lang="yaml">
+en:
+  title: 'Purple Friday 2024'
+  subtitle: 'What is happening on Purple Friday this year? Find an overview here:'
+nl:
+  title: 'Paarse Vrijdag 2024'
+  subtitle: 'Wat is er dit jaar op Paarse Vrijdag? Zie hier een overzicht (alleen in het Engels):'
+</i18n>
+
+<script setup>
+import { ref, watchEffect } from 'vue'
+const { t } = useT()
+
+const faculties = (await useAsyncData(() => queryContent('resources').find())).data
+const active = ref(0)
+
+function setActive(index) {
+  active.value = index
+}
+
+watchEffect(() => {
+  faculties.value.sort((a, b) => {
+    return (a.order || 0) - (b.order || 0)
+  })
+})
+</script>
```

Comment:
Ik zou het volgende nog toevoegen:
```
const activeFaculty = computed(() => faculties.value[active.value])
```

dan kun je hieronder in het component bijv  `activeFaculty.name` doen ipv `faculties[active].name`